pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/codestyle/xml/MethodNamingConventions.xml

```diff
@@ -68,8 +68,8 @@ public class Foo {
     
     <test-code>
         <description>#1573 method names should not contain underscores, but skip test methods 1</description>
-        <expected-problems>0</expected-problems>
         <rule-property name="skipTestMethodUnderscores">true</rule-property>
+        <expected-problems>0</expected-problems>
```

Comment:
I don't understand this change: Why is this necessary, or did we change the rule-test schema?

Reply:
The builds failed here because the order was wrong. I have no clue why this didn’t fail in the past but I fixed it for this pr

Reply:
We seem to run into this problem: https://github.com/junit-team/junit5/issues/987

Reply:
#1707